nomos-libp2p/src/config.rs

```diff
@@ -142,7 +142,7 @@ impl From<GossipsubConfigDef> for gossipsub::Config {
     }
 }
 
-mod secret_key_serde {
+pub mod secret_key_serde {
```

Comment:
This are the kind of things I warn about. Everytime something we want to change we will have to bridge it first. There is no need.